versions/1.0.0.md

```diff
@@ -480,6 +487,91 @@ criteria:
     - $statusCode == 503
 ```
 
+#### Component Object
+
+Holds a set of reusable objects for different aspects of the Workflows Specification. All objects defined within the components object will have no effect on the workflow unless they are explicitly referenced from properties outside the components object.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---|---
+<a name="componentInputs"></a> inputs | Map[`string`, [Schema Object](#schemaObject)] | An object to hold reusable JSON Schema objects to be referenced from workflow inputs.
+<a name="componentParameters">parameters</a> | Map[string, [Parameter Object](#parameter-object)] | An object to hold reusable Parameter Objects
+
+This object MAY NOT be extended with [Specification Extensions](#specificationExtensions).
+
+All the fixed fields declared above are objects that MUST use keys that match the regular expression: `^[a-zA-Z0-9\.\-_]+$`. The key is used to refer to the input or parameter in other parts of the Workflow document.
+
+Field Name Examples:

```

Comment:
I'm not sure what the field name examples are representing here. We don't have this in other sections

Reply:
It's just a regex for the keys in the Maps - happy to remove this section but I figured it'd be a good idea.